### DISCUSSION TITLE
Disable SDRTrunk Audio

### DISCUSSION THREAD

**Discussion by hoffadg** [chuot/rdio-scanner#309] [Q&A]
I have SDRTrunk running on the same machine as rdio-scanner.  I'm hearing audio from both SDRTrunk and rdio-scanner.  Has anyone figured out how to permanently mute SDRTrunk audio?  There is a button in the GUI to do it but it would be nice if it could just be disabled every time it is run.

**Comment by chuot** [comment]
I think you can do it within SDRTrunk on the aliases tab where for each alias you have a record and a play checkbox, just uncheck the play checkbox and leave the record one checked.
